Project-Euler/Problem026.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Problem - Longest Recurring Cycle
```

Comment:
I'd merge this comment with the documentation of the function below: Just explain which problem the function solves.

Reply:
my function answers problem026